elixir-format.el

```diff
@@ -125,54 +125,76 @@ Shamelessly stolen from go-mode (https://github.com/dominikh/go-mode.el)"
               (error "Invalid rcs patch or internal error in elixir-format--apply-rcs-patch"))))))))
   )
 
+(defun elixir-mix-path-set? ()
+  (and (not (= (length elixir-format-mix-path) 0)) (not (= (length elixir-format-elixir-path) 0)))
+  )
+
+(defun display-configuration-missing-error ()
+  (let ((errbuff (get-buffer-create "*elixir-format-configuration-missing-errors*")))
+    (progn
+      (with-current-buffer errbuff
+        (setq buffer-read-only nil)
+        (erase-buffer)
+        (insert "Please check https://github.com/elixir-editors/emacs-elixir#elixir-format docs to customize necessary variables for elixir and mix.")
+        (setq buffer-read-only t)
+        (ansi-color-apply-on-region (point-min) (point-max))
+        (special-mode))
+
+      (if is-interactive
```

Comment:
This variable does not exist in this function, so I don't think this code will work.

Reply:
done.